src/Components/Delay.js

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect, useState } from "react";
+
```

Comment:
I think I see whats happening here, but would be awesome to go over this

Reply:
Yes, please!